persist/schema/types.go

```diff
@@ -25,14 +25,29 @@ package schema
 // tooling needs to upgrade older files to newer files before a server restart
 const MajorVersion = 1
 
+// FilesetFileType is an enum that indicates what type of file the fileset files
+// on disk represent.
+type FilesetFileType int64
```

Comment:
any reason to avoid re-using the enum from `persist/types.go`?

Reply:
nope, I just made this one first and then added the other one later and forgot about this. Will refactor

Reply:
done